subsys/mpsl/cx/thread/mpsl_cx_thread.c

```diff
@@ -165,6 +170,12 @@ static int32_t register_callback(mpsl_cx_cb_t cb)
 {
 	callback = cb;
 
+	if (cb != NULL) {
+		nrfx_gpiote_trigger_enable(grant_abs_pin, true);
+	} else {
+		nrfx_gpiote_trigger_disable(grant_abs_pin);
+	}
```

Comment:
It's OK, just leaving a comment to document.
The reason why `gpio_pin_interrupt_configure_dt` is not called here but lower level api is uset is that the `register_callback` function is used in time critical procedures that cannot tolerate so time consuming call like `gpio_pin_interrupt_configure_dt`.